src/components/QuickReplies.tsx

```diff
@@ -52,7 +61,7 @@ function renderButton(
     const innerHTML = TextUtil.processText(title)
     return (
       <div
-        className="button"
+        className={`button ${operatorConversationId ? 'disabled' : null}`}
```

Comment:
Instead of checking operatorConversationId everywhere in each control, it would have been easier to disable it on a global level using pure CSS, e.g. by adding a class to the top-level `.botsi-web-client` div, like I did it in the studio:

```
.botsi-web-client.block-input {
    div.button,
    button,
    a,
    input {
      pointer-events: none !important;
    }
  }
```

This way you only need to do it on one place instead of everywhere in each input component.

Reply:
This doesn't disable everything, It only disables Quick Replies. I also disable close buttons,Single/Multi Item picker,Forms,Datepickers etc...